react/features/invite/components/add-people-dialog/native/AddPeopleDialog.js

```diff
@@ -421,17 +421,17 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
         );
     }
 
-    _renderInvitedItem: Object => ?React$Element<*>
+    _renderInvitedItem: Object => React$Element<any> | null
```

Comment:
Doesn't `?React$Element<any>` work?

Reply:
No, the point was the ```null```, because ```?``` means ```null or undefined``` which is not suitable for an expected ```null``` value. I just updated the ```*``` to ```any``` due to an unrelated warning.